basketball_reference_web_scraper/output/columns.py

```diff
@@ -20,6 +20,7 @@
     "blocks",
     "turnovers",
     "personal_fouls",
+    "plus_minus",
```

Comment:
@benjaminmesser I think we need to be a little careful here as the `PLAYER_SEASON_BOX_SCORE_COLUMN_NAMES` uses `SHARED_COLUMN_NAMES` and already has `plus_minus` as a column.

https://github.com/jaebradley/basketball_reference_web_scraper/blob/84d6c251095f906f6af4676cdcc3603073e34181/basketball_reference_web_scraper/output/columns.py#L28

IMHO, I think the most backwards-compatible thing that we can do is something like this

```python
SHARED_COLUMN_NAMES = [
    "team",
    "location",
    "opponent",
    "outcome",
    "seconds_played",
    "made_field_goals",
    "attempted_field_goals",
    "made_three_point_field_goals",
    "attempted_three_point_field_goals",
    "made_free_throws",
    "attempted_free_throws",
    "offensive_rebounds",
    "defensive_rebounds",
    "assists",
    "steals",
    "blocks",
    "turnovers",
    "personal_fouls",
    "game_score",
]

BOX_SCORE_COLUMN_NAMES = ["slug", "name"] + SHARED_COLUMN_NAMES + ["plus_minus"]

PLAYER_SEASON_BOX_SCORE_COLUMN_NAMES = ["active", "date", "points_scored", "plus_minus"] + SHARED_COLUMN_NAMES
```

It's definitely less satisfying of a solution, but the main rationale is that all previous column orderings are backwards-compatible.

If there aren't any strong objections, feel free to make this change and I'll be happy to merge and publish a minor version update with this additional data field.

Reply:
Is there a reason why it should be done that way instead of just adding "plus_minus" to `SHARED_COLUMN_NAMES` and removing it from the `PLAYER_SEASON_BOX_SCORE_COLUMN_NAMES` line? Is there something else that uses `SHARED_COLUMN_NAMES` other than `BOX_SCORE_COLUMN_NAMES` and `PLAYER_SEASON_BOX_SCORE_COLUMN_NAMES`?

Something like this:

```
SHARED_COLUMN_NAMES = [
    "team",
    "location",
    "opponent",
    "outcome",
    "seconds_played",
    "made_field_goals",
    "attempted_field_goals",
    "made_three_point_field_goals",
    "attempted_three_point_field_goals",
    "made_free_throws",
    "attempted_free_throws",
    "offensive_rebounds",
    "defensive_rebounds",
    "assists",
    "steals",
    "blocks",
    "turnovers",
    "personal_fouls",
    "game_score",
    "plus_minus",
]

BOX_SCORE_COLUMN_NAMES = ["slug", "name"] + SHARED_COLUMN_NAMES

PLAYER_SEASON_BOX_SCORE_COLUMN_NAMES = ["active", "date", "points_scored"] + SHARED_COLUMN_NAMES
```

Reply:
Yeah, this is something I considered and a fair alternative approach. My reasoning for not taking this approach is that it changes the order of the `PLAYER_SEASON_BOX_SCORE_COLUMN_NAMES` (for those that export data via CSV) - `plus_minus` is no longer the fourth column, but no the penultimate column.

For most people, using most sane CSV parsing libraries, that allow for parsing by column names, this _shouldn't _ be problematic.

However, there may be some users where the ordering of columns matter, and my initial proposal, from my perspective, keeps the most backwards-compatible behavior.

Thoughts?

Reply:
That's a good point. People probably really shouldn't be indexing by column number in general, but you're right that if anyone were to do that, adding to SHARED_COLUMN_NAMES would break any hard-coded checks like that. If you're going for maximal backwards compatibility then I think your solution makes the most sense. I've added another commit to this pull request to reflect your suggestion.